assets/src/platform-admin/pages/RestrictedPage.tsx

```diff
@@ -1,16 +1,17 @@
 import * as React from "react";
-import { ErrorAlert } from "@pay/web";
-
-import { usePayUser } from "../../users";
+import { ErrorAlert, Loader } from "@pay/web";
+import { useAuth0 } from "../../auth/AuthContext";
 
 interface Props {
   children: JSX.Element;
 }
 
 const RestrictedPage: React.FC<Props> = ({ children }) => {
-  const user = usePayUser();
+  const { isInitializing, isAuthenticated } = useAuth0();
+
+  if (isInitializing) return <Loader message="Loading " />;
 
-  if (user && user.platformAdmin) {
```

Comment:
@acodegirl this restricted page component is inside the `platform-admin` app. by changing this condition, are you saying that any user is allowed to access platform admin pages if they are authenticated? or is the check for `user.platformAdmin === true` occurring elsewhere now?